app/static/js/stories/search.js

```diff
@@ -28,20 +30,28 @@ $(function () {
                 start: startInput.val()
             },
             success: function (data) {
+                noMore.hide();
                 if (data.total !== 0 && !append) {
                     noResultsFound = false;
+                    loadMore.show();
                     results.html(data.results);
                     start = start + data.count;
                     total = data.total;
-                }
-                else if (data.total !== 0 && append ) {
+                } else if (data.total !== 0 && append) {
                     noResultsFound = false;
+                    loadMore.show();
                     results.append(data.results);
                     start = start + data.count;
-                }
-                else {
+                    if (data.count === 0) {
+                        loadMore.hide();
+                        noMore.show();
+                    } else {
+                        loadMore.show();
+                    }
+
+                } else {
                     noResultsFound = true;
```

Comment:
Edge case, hide the load more button when no results are found.

`loadMore.hide();`

Reply:
done